### PR TITLE
Restructure Code

### PR DESCRIPTION
**Pull request by MatthewFrisby**
# Description
Restructure Code to better keep organization for future additions to the codebase

## Related US/Task
N/A

## Motivation and Context
Having all of the code in one file was unsustainable

## How Has This Been Tested
Compiled and build code to test functionality of procedures and functions and everything still works the same

## Types of changes

- [x] Refactoring / dependency upgrade / docs change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Non-functional requirement

## Visual screenshots
N/A

### CODE REVIEW

File: cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    public Node getNode() {
+        return node;
+    }
+
+    public ArrayList<FhirRelationship> getRelationships() {
+        return relationships;
+    }
+
+    public void setNode(Node node) {
```
Comment (DMadrid) [inline cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java:20]: If we're sticking with Java, we should consider Lombok for a future task

File: cyfhir/src/main/java/com/Optum/CyFHIR/procedures/toStream.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            rProps = filterProperties(rProps, relFilters.get(typeName));
+        }
+        rProps.forEach((k, v) -> mMap.put(prefix + k, v));
+        return mMap;
+    }
+
+    private static Map<String, Object> filterProperties(Map<String, Object> props, List<String> filters) {
+        boolean isExclude = filters.get(0).startsWith("-");
+
+        return props.entrySet().stream().filter(e -> isExclude ? !filters.contains("-" + e.getKey()) : filters.contains(e.getKey())).collect(Collectors.toMap(k -> k.getKey(), v -> v.getValue()));
```
Comment (DMadrid) [inline cyfhir/src/main/java/com/Optum/CyFHIR/procedures/toStream.java:75]: Can we improve the readability of this line

File: cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java (hunk 1 above)
Comment (MatthewFrisby) [reply cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java:20]: That is a good idea!

File: cyfhir/src/main/java/com/Optum/CyFHIR/procedures/toStream.java (hunk 2 above)
Comment (MatthewFrisby) [reply cyfhir/src/main/java/com/Optum/CyFHIR/procedures/toStream.java]: Should be better now

File: cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java (hunk 1 above)
Comment (tyleryam) [reply cyfhir/src/main/java/com/Optum/CyFHIR/models/FhirRecursiveObj.java:20]: Agreed, Lombok would reduce a lot of boilerplate in Java.

**Pull request merged**